content/02.main-text.md

```diff
@@ -47,23 +47,23 @@ Although they work well for editing text, they lack sufficient features for mana
 | Editing software | Any text editor | Web interface | Web interface | Web interface | Web interface | Any text editor |
 | Document format | Markdown | LaTeX | LaTeX | Proprietary | Proprietary | Markdown |
 | Templating | Yes | Yes | Yes | No | No | No |
-| Technial expertise required | Yes | Yes | Yes | No | No | Yes |
+| Technical expertise required | Yes | Yes | Yes | No | No | Yes |
 
 Table: Collaborative writing platforms.
 A summary of features that differentiate Manubot from existing collaborative writing platforms.
 We assessed features on June 15, 2018 using the free version of each platform.
 Some platforms offer additional features through a paid subscription or software.
 <sup>1</sup>Additional functionality, such as bibliography management, is available by editing the Word document stored in OneDrive with the paid Word desktop application.
 <sup>2</sup>Conversations about modifications take place on the document as comments, annotations, or unsaved chats.
-There is no separate integrated forum for discussing and editing revisions.
-<sup>3</sup>Simultaneous edits online can be attributed to a single author in a git commit.
+There is no integrated forum for discussing and editing revisions.
+<sup>3</sup>In some circumstances, a git commit containing different edits made at the same time by distinct authors may be attributed to a single author.
```

Comment:
@dhimmel wrote https://github.com/agitter/meta-review/pull/1#discussion_r206364776
> Isn't the Overleaf issue worse? Are we sure any commits are correctly attributed to their author?

> I think it should be more like, "git commits created by Overleaf do not correspond to modular revisions and do not retain accurate authorship information."

The Overleaf documentation I copied in https://github.com/greenelab/meta-review/issues/27#issuecomment-407889342 details how they use git commits (in the v1 interface).  The commit author is complicated.  I believe it depends on whether the last editor before a commit is triggered is logged in or not.  The edits do not actually have to be made at the same time to confound attribution.  They can be combined in a single commit if they are made before a commit-generating event.

Proposed phrasing:
> In some circumstances, Overleaf git commits are not modular.  Edits made by distinct authors may be attributed to a single author.